app/src/main/java/com/yeonae/chamelezone/view/course/CourseDetailActivity.kt

```diff
@@ -1,18 +1,90 @@
 package com.yeonae.chamelezone.view.course
 
 import android.os.Bundle
+import android.util.Log
+import android.view.View
 import androidx.appcompat.app.AppCompatActivity
 import com.yeonae.chamelezone.Injection
 import com.yeonae.chamelezone.R
+import com.yeonae.chamelezone.ext.Url.IMAGE_RESOURCE
+import com.yeonae.chamelezone.ext.glideImageSet
 import com.yeonae.chamelezone.network.model.CourseResponse
 import com.yeonae.chamelezone.view.course.presenter.CourseDetailContract
 import com.yeonae.chamelezone.view.course.presenter.CourseDetailPresenter
 import kotlinx.android.synthetic.main.activity_course_detail.*
 
 class CourseDetailActivity : AppCompatActivity(), CourseDetailContract.View {
-    override fun showCourseDetail(course: CourseResponse) {
-        tv_course_title.text = course.title
-        tv_course_content.text = course.content
+    override fun showCourseDetail(course: List<CourseResponse>) {
```

Comment:
```suggestion
    override fun showCourseDetail(courseList: List<CourseResponse>) {
```
단일 객체를 의미하는 변수명을 변경하는 것이 좋을 것 같아요.
또 empty일때의 처리가 없는 것 같아요

Reply:
courseList가 empty일때를 말하는거죠?!

Reply:
네 맞아요